### ISSUE TITLE
Failure to correct poetry paths after update

### ISSUE THREAD

**Issue by FroggyFlox** [rockstor/rockstor-core#2757]
On my personal Rockstor machine, I recently updated from a pre-Poetry version (4.1.0) to latest Testing (5.0.5). Despite the shameful failure to keep that machine up-to-date until now, the update went without a hiccup.
Since then, however, I did receive an email notification that my scheduled task failed:
```
/bin/bash: /opt/rockstor//bin/st-pool-scrub: No such file or directory
```

This look exactly like what was supposed to be corrected by #2558: migrating previous path to poetry venv. After a closer look, we can see that my previous (currently failing) path contains a double `//` (not sure why). This is why our mechanism to update to poetry paths implemented in #2558 failed. We are indeed looking for the presence of the full path with only single `/` and thus fail to match that pattern:
```python
>>> import re
>>> pattern = "/opt/rockstor/bin/"
>>> line="42 3 * * 5 root /opt/rockstor//bin/st-pool-scrub 1 \*-*-*-*-*-*"
>>> re.search(pattern, line)
>>>
```

We may want to update that pattern to:
```python
>>> pattern = "/opt/rockstor[/]+bin/"
>>> re.search(pattern, line)
<re.Match object; span=(16, 35), match='/opt/rockstor//bin/'>
```

**Comment by FroggyFlox** [comment]
To do:

- [ ] Add unit test for `system.osi.replace_pattern_inline()` and verify it can correctly treat instances of `/`
- [ ] Update pattern used as detailed above and verify test still passes

**Comment by phillxnet** [comment]
Closing as:
Fixed by #2771
